pkg/registry/apis/provisioning/webhook.go

```diff
@@ -65,6 +66,10 @@ func (s *webhookConnector) Connect(ctx context.Context, name string, opts runtim
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		logger := logging.FromContext(r.Context()).With("logger", "webhook-connector", "repo", name)
 		ctx := logging.Context(r.Context(), logger)
+		if !s.webhooksEnabled {
+			responder.Error(errors.NewBadRequest("webhooks are not enabled"))
```

Comment:
I'm not convinced this is accurate -- there a LOTS of ways to get things setup so that this webhook routes correctly without having grafana manage the public URL

Reply:
ahh -- but then we could not create the webhook for them 🤔 so maybe it really does not matter